doc/users/whats_new/errorbar_offsets.rst

```diff
@@ -0,0 +1,10 @@
+Errorbar plots can shift which points have error bars
+----------------------
+
+Previously, `plt.errorbar()` accepted a kwarg `errorevery` such that the
+command `plt.errorbar(x, y, yerr, errorevery=6)` would add error bars to
+datapoints `x[::6], y[::6]`.
+
+`errorbar()` now also accepts a tuple for `errorevery` such that
+`plt.errorbar(x, y, yerr, errorevery= (a, b) )` adds error bars to points
+`x[b%a::a], y[b%a::a]`.
```

Comment:
See also the markevery parameter (http://matplotlib.org/api/lines_api.html#matplotlib.lines.Line2D.set_markevery, http://matplotlib.org/examples/pylab_examples/markevery_demo.html), which inverts the two args.  Even if you do not support all the variations that markevery does, it would be nice to make sure the cases that *are* supported (i.e., `(a, b)` right now) are consistent between the two.

Extra points if the machinerey of markevery can be somehow reused...

Reply:
Should be good now